cypress/e2e/buy-shirt.cy.ts

```diff
@@ -0,0 +1,33 @@
+describe("Buy a t-shirt", () => {
+  it("then the t-shirt should be bought", () => {
+    cy.visit("http://automationpractice.com/");
+    cy.get("#block_top_menu > ul > li:nth-child(3) > a").click();
+    cy.get(
+        "#center_column a.button.ajax_add_to_cart_button.btn.btn-default",
+    ).click();
+    cy.get("[style*='display: block;'] .button-container > a").click();
+    cy.get(".cart_navigation span").click();
+
+    cy.get("#email").type("aperdomobo@gmail.com");
+    cy.get("#passwd").type("WorkshopProtractor");
+
+    // Debes completar la prueba ...
+    // Step 8
+    cy.get("#SubmitLogin").click();
+    // Step 9
+    cy.get("[name='processAddress']").click();
+    // Step 10
+    cy.get("#uniform-cgv").click();
+    // Step 11
+    cy.get("[name='processCarrier']").click();
+    // Step 12
+    cy.get(".bankwire").click();
+    // Step 13
```

Comment:
Please delete those comments, are not useful and makes the code not pretty readable